cfpb/cfgov-refresh
# Mega menu: Add Payday Loans link

Add `Payday Loans` link to Consumer Tools vertical in mega menu

## Additions

- Payday Loans link


## Testing

1. Check out branch
2. Verify that Payday Loans link appears in mega menu under Consumer Tools & goes to Ask category page

## Screenshots
<img width="1002" alt="screen shot 2017-09-18 at 10 13 51 am" src="https://user-images.githubusercontent.com/778171/30554749-70edbcd2-9c5a-11e7-9bbf-c5a30b3e287d.png">



## Checklist
